casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -64,6 +62,7 @@ class MultiParentCasperImpl[F[_]: Sync: Concurrent: ConnectionsCell: TransportLa
   import MultiParentCasper.MetricsSource
 
   implicit private val logSource: LogSource = LogSource(this.getClass)
+  private[this] val F                       = Sync[F]
```

Comment:
`syncF` for consistency?

Reply:
Yeah - as you wish :smile:

Reply:
Fixed in 142b3c680bdb5116c371380a004ad6004c81d6eb